tests/test_interface.rb

```diff
@@ -958,6 +959,44 @@ def test_interface_ipv4_arp_timeout
     assert_raises(RuntimeError) { nonvlanint.ipv4_arp_timeout = 300 }
   end
 
+  def test_interface_fabric_forwarding_anycast_gateway
+    # Setup
+    config('no interface vlan11')
+    int = Interface.new('vlan11')
+    config('feature fabric forwarding')
+    foo = VxlanGlobal.new
+    foo.anycast_gateway_mac = '1223.3445.5668'
```

Comment:
@glennmatthews It is needed in `I2` images.
```
agent-lab10-nx(config)# sh run | i 'feature fabric'
agent-lab10-nx(config)# conf t
agent-lab10-nx(config)# int vlan 11
agent-lab10-nx(config-if)# fabric fo?
                           ^
% Invalid command at '^' marker.
agent-lab10-nx(config-if)# 
agent-lab10-nx(config-if)# feature fabric forwarding
agent-lab10-nx(config)# int vlan 11
agent-lab10-nx(config-if)# fabric forwarding mode anycast-gateway 
```
vs. the behavior on `I3` images...

```
dt-n9k5-1(config)# feature interface-vlan 
dt-n9k5-1(config)# 
dt-n9k5-1(config)# sh run | i 'feature fabric'
dt-n9k5-1(config)# 
dt-n9k5-1(config)# int vlan 11
dt-n9k5-1(config-if)# fabric forwarding mode anycast-gateway 
dt-n9k5-1(config-if)# 
```